### PR TITLE
Fixed issue where items were being incorrectly positioned in the toolbar

### PR DESCRIPTION
**Pull request by nwalters512**

### CODE REVIEW

File: android/src/main/java/com/thebluealliance/androidclient/activities/HomeActivity.java [hunk 1]
```diff
@@ -55,7 +55,8 @@
 
     private static final String MAIN_FRAGMENT_TAG = "mainFragment";
 
-    @Inject TBAStatusController mStatusController;
+    @Inject
+    TBAStatusController mStatusController;
```
Comment (phil-lopreiato) [inline android/src/main/java/com/thebluealliance/androidclient/activities/HomeActivity.java]: Keep this on one line, per Android Style guide.

You can import it into Android Studio with the instructions [here](https://github.com/the-blue-alliance/the-blue-alliance-android#code-style)

**Comment by phil-lopreiato** [comment]
:+1: after style fix

File: android/src/main/java/com/thebluealliance/androidclient/activities/HomeActivity.java (hunk 1 above)
Comment (nwalters512) [inline android/src/main/java/com/thebluealliance/androidclient/activities/HomeActivity.java]: Code -> Reformat Code doesn't seem to be fixing this, isn't that the whole point of defining a code style?

Comment (nwalters512) [inline android/src/main/java/com/thebluealliance/androidclient/activities/HomeActivity.java]: Moreover, Reformat Code actually sticks this back on a different line, even after applying the proper code style.

Comment (phil-lopreiato) [inline android/src/main/java/com/thebluealliance/androidclient/activities/HomeActivity.java]: I just pushed a fix in https://github.com/the-blue-alliance/the-blue-alliance-android/commit/90e5e6642492ae86493a55948dcee8e4634394ab, I realized I forgot to set the property. Copy over the new file from master

**Pull request merged**